foosball/games/models.py

```diff
@@ -62,3 +65,94 @@ def __str__(self):
             self.score,
             ", ".join(map(str, self.players.all()))
         )
+
+
+class Player(models.Model):
```

Comment:
What would make sense imho would be to foreign key `Team` to `Player`, not `User` :) But that can be done in some future data migration if wanted.